HCore-Emailing/HCore-Emailing.csproj

```diff
@@ -7,10 +7,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="MailKit" Version="4.4.0" />
+    <PackageReference Include="MailKit" Version="4.7.1" />
```

Comment:
[Release notes](https://github.com/jstedfast/MailKit/blob/master/ReleaseNotes.md)
I was able to test the latest version with using smtp.gmail.com
No breaking changes as far as I can see